pythonAPIClient/client.py

```diff
@@ -391,3 +391,40 @@ def search_camera(self, latitude=None, longitude=None, radius=None, camera_type=
         for current_object in camera_response_array:
             camera_processed.append(Camera.process_json(**current_object))
         return camera_processed
+
+    def get_change_log(self, start=None, end=None, offset=None):
+        """
+        Parameters
+        ----------
+        start : str, optional
+            Start time of the log user desires to query
+        end : str, optional
+            End time of the log user desires to query
+        offset : str, optional
+            How many logs to skip
+
+        Returns
+        -------
+            A list of objects containing cameraID and creation time of the log.
```

Comment:
according to https://python-sprints.github.io/pandas/guide/pandas_docstring.html#parameter-types , can we use `list of dict of {str: str}` to specify return type?

Reply:
add errors raises also